<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisCorrelation</class>
 <widget class="QWidget" name="analysisCorrelation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Correlation Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ChartView" name="plot">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pearson</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Shared genes</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="pearson">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Pearson correlation value</string>
         </property>
         <property name="statusTip">
          <string>Pearson correlation value</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="sharedGenes">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Number of genes shared between selections</string>
         </property>
         <property name="statusTip">
          <string>Number of genes shared between selections</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Spearman</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBoxNormalization">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Different normalization methods for the counts</string>
         </property>
         <property name="statusTip">
          <string>Normalization methods</string>
         </property>
         <property name="whatsThis">
          <string>Different normalization methods for the counts</string>
         </property>
         <property name="title">
          <string>Normalization</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="normalization_raw">
            <property name="text">
             <string>RAW</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="normalization_rel">
            <property name="text">
             <string>REL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="normalization_scran">
            <property name="text">
             <string>SCRAN</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="normalization_tpm">
            <property name="text">
             <string>TPM</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="normalization_deseq">
            <property name="text">
             <string>DESeq2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="logScale">
            <property name="text">
             <string>Log scale</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="spearman">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Spearman correlation value</string>
         </property>
         <property name="statusTip">
          <string>Spearman correlation value</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="exportPlot">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>SplitHCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Export the correlation plot to a file</string>
         </property>
         <property name="statusTip">
          <string>Export the correlation plot to a file</string>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>analysis/ChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
